src/sentry/dynamic_sampling/models/common.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+from typing import Any, List, Optional, Union
+
+import sentry_sdk
+
+from sentry.dynamic_sampling.models.base import Model, ModelInput
+from sentry.dynamic_sampling.rules.utils import ProjectId, TransactionName
+
+
+@dataclass
+class ModelClass:
```

Comment:
I would try to avoid using the word "class" in a class name as much as possible. Judging by its usage, `RebalancedItem` (maybe with a comment explaining that item can be a project or a transaction) would be more appropriate.